steamed_hams.py

```diff
@@ -86,7 +86,7 @@
     mode = "debug"
     data_write_args = "--file-type=hdf5 --enable-raw-acfs --enable-bfiq --enable-antenna-iq"
 elif sys.argv[2] == "pythonprofiling":
-    python_opts = "-O -u -m cProfile -o testing/python_testing/{module}.cprof"
+    python_opts = "-O -u -m cProfile -o testing/python_testing/{module}.cprof"  # TODO: Does this need updating to tools? I don't see the python_testing directory
```

Comment:
this may be out of date but it's not a big concern to me as it's just specifying output file location, and I don't think that location needs to be part of the git repo.